Signal/src/view controllers/CallViewController.swift

```diff
@@ -287,21 +516,25 @@ class CallViewController: UIViewController {
         self.dismiss(animated: true)
     }
 
-    @IBAction func didPressMute(sender muteButton: UIButton) {
+    func didPressMute(sender muteButton: UIButton) {
         Logger.info("\(TAG) called \(#function)")
         muteButton.isSelected = !muteButton.isSelected
         CallService.signalingQueue.async {
```

Comment:
FYI I touched these methods in https://github.com/WhisperSystems/Signal-iOS/pull/1587/files#diff-7b6a32382aa648492d9e501eb1e6269cR286

If there *is* a conflict it *should* be pretty straight forward to resolve. LMK if you have an issue.